go.mod

```diff
@@ -6,7 +6,7 @@ go 1.19
 replace (
 	github.com/onsi/ginkgo/v2 => github.com/onsi/ginkgo/v2 v2.9.1
 	github.com/onsi/gomega => github.com/onsi/gomega v1.27.4
```

Comment:
wondering if these two also need update with that?

Reply:
Yes, `1.4.1 capi` uses those versions so should be fine.

Reply:
Done.